### PR TITLE
Add New Edge device

### PR DESCRIPTION
**Pull request by isaac73**
# Type of change? 

- [ ] Bug fix
- [x] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Add New Edge device
...

**Checklist:**

- [x] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-remote-monitoring-webui/1135)

### CODE REVIEW

File: src/components/pages/devices/flyouts/deviceNew/deviceNew.js [hunk 1]
```diff
@@ -251,6 +268,30 @@ export class DeviceNew extends LinkedComponent {
...
+      (value === 'true') ? Config.device.edgeDevice : Config.device.device));
+    if (value === 'true') {
+      this.setState({
+        formData: {
+          ...this.state.formData,
+          isEdgeDevice: true,
+          isSimulated: false
+        }
+      });
+    }
```
Comment (spryor) [inline src/components/pages/devices/flyouts/deviceNew/deviceNew.js]: `} else {`

File: src/components/pages/devices/flyouts/deviceNew/deviceNew.js [hunk 2]
```diff
@@ -362,14 +404,25 @@ export class DeviceNew extends LinkedComponent {
...
+                  <Radio link={this.deviceLink} value={deviceOptions.device.value} onChange={this.deviceChange}>
+                    {t(deviceOptions.device.labelName)}
                   </Radio>
-                  <Radio link={this.deviceTypeLink} value={deviceTypeOptions.physical.value} onChange={this.deviceTypeChange}>
-                    {t(deviceTypeOptions.physical.labelName)}
+                  <Radio link={this.deviceLink} value={deviceOptions.edgeDevice.value} onChange={this.deviceChange}>
+                    {t(deviceOptions.edgeDevice.labelName)}
                   </Radio>
                 </FormGroup>
+                {isEdgeDevice &&
```
Comment (spryor) [inline src/components/pages/devices/flyouts/deviceNew/deviceNew.js]: newline
```js
{
   isEdgeDevice &&
```

File: src/components/pages/devices/flyouts/deviceNew/deviceNew.js (hunk 1 above)
Comment (isaac73) [reply src/components/pages/devices/flyouts/deviceNew/deviceNew.js]: fixed

File: src/components/pages/devices/flyouts/deviceNew/deviceNew.js (hunk 2 above)
Comment (isaac73) [reply src/components/pages/devices/flyouts/deviceNew/deviceNew.js]: fixed

**Pull request merged**